sdks/python/test-suites/xlang/build.gradle

```diff
@@ -58,5 +58,19 @@ def gcpXlangCommon = new CrossLanguageTaskCommon().tap {
 }
 xlangTasks.add(gcpXlangCommon)
 
+def ioExpansionProject = project.project(':sdks:java:io:expansion-service')
+
+def ioXlangCommon = new CrossLanguageTaskCommon().tap {
+    name = "ioCrossLanguage"
+    expansionProjectPath = ioExpansionProject.getPath()
+    collectMarker = "uses_io_expansion_service"
+    startJobServer = setupTask
+    cleanupJobServer = cleanupTask
+    //Temporary hardcoding while we await GMK (google managed kafka) TODO johnjcasey
+    //See .test-infra/kafka/bitnami/README.md for setup instructions
+    additionalArgs = ["KAFKA_BOOTSTRAP_SERVER":"10.128.0.40:9094,10.128.0.28:9094,10.128.0.165:9094"]
```

Comment:
consider making this configurable, could use sth like `project.findProperty('kafkaBootstrapServer') ?: "10.128.0.40:9094,10.128.0.28:9094,10.128.0.165:9094"`, or alternatively, set this in test CI (jenkins groovy file or github action yml) so we do not need to add TODO here